backend/pah_fm/urls.py

```diff
@@ -15,7 +15,10 @@
 """
 from django.contrib import admin
 from django.urls import path
+from ..fleet_management.api import PassengerListView, CarListView
 
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('api/users/passengers', PassengerListView.as_view(), name='passengers'),
+    path('api/users/cars', CarListView.as_view(), name='cars'),
```

Comment:
Why not `api/passengers` and `api/cars`? I think that passengers and cars are entities on their own.